builtin/logical/pki/path_root.go

```diff
@@ -121,6 +121,7 @@ func (b *backend) pathCADeleteRoot(ctx context.Context, req *logical.Request, _
 	// explicitly versus deleting everything.
 	resp := &logical.Response{}
 	resp.AddWarning("DELETE /root deletes all keys and issuers; prefer the new DELETE /key/:key_ref and DELETE /issuer/:issuer_ref for finer granularity, unless removal of all keys and issuers is desired.")
+	go logical.SendEvent(ctx, b, "pki/ca-delete-root")
```

Comment:
It looks like it would be pretty straightforward to add the IDs to the response and event. Not sure about doing the name without doing another dip into storage, though. But I can add to this PR the IDs at least.